google_photos_takeout_helper/__main__.py

```diff
@@ -104,21 +116,29 @@ def for_all_files_recursive(
                 print('Found something weird...')
                 print(file)
 
-
     def is_photo(file):
         what = _os.path.splitext(file.lower())[1]
         if what not in photo_formats:
             return False
+        # skips the extra photo file, like edited or effects. They're kinda useless.
+        if args.skip_extras or args.skip_extras_harder:  # if the file name includes something under the extra_formats, it skips it.
+            for extra in extra_formats:
+                if extra in file.lower():
+                    return False
+        if args.skip_extras_harder:
+            search = '\([0-9]\)\.'
```

Comment:
Any digit, including 13, 123 etc
```suggestion
            search = '\(\d+\)\.'
```